client/signup/main.jsx

```diff
@@ -329,6 +331,11 @@ const Signup = React.createClass( {
 			null;
 	},
 
+	pageTitle() {
+		const accountFlowName = 'account';
+		return this.props.flowName === accountFlowName ? translate( 'Create an account' ) : translate( 'Create a site' );
```

Comment:
Should this always fall back to "Create a site"? Not sure if there's anything else `props.flowName` could be, but there may be in the future.